modules/lib/api/document.xql

```diff
@@ -2,8 +2,8 @@ xquery version "3.1";
 
 module namespace dapi="http://teipublisher.com/api/documents";
 
-import module namespace router="http://exist-db.org/xquery/router";
-import module namespace errors = "http://exist-db.org/xquery/router/errors";
+import module namespace router="http://e-editiones.org/roaster/router";
```

Comment:
curious there seems to be a display problem, line 5 reads `import module namespace router="http://e-editiones.org/roaster";`